proto/exocore/avs/tx.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+package exocore.avs;
+import "cosmos/msg/v1/msg.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/ExocoreNetwork/exocore/x/avs/types";
+
+// AVSinfo represent the information of avs
+message AVSInfo {
+  // name of avs
+  string name = 1;
+  // address of avs
+  string avs_address = 2;
+  // registered operator of avs
+  repeated string operator_address = 3;
+  // the owner who has permission for avs
+  string avs_owner_address = 4;
+  // asset_basic_info is all the basic asset information of the avs.
+  repeated string asset_id = 5;
```

Comment:
what about minimum operator-level self-delegations?

Reply:
Will be handled in operator&delegation module I think

Reply:
okay to leave as a TODO for the next phase.